cmd/ketch/builder.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"io"
+
+	"github.com/spf13/cobra"
+)
+
+const builderCmdHelp = `
+Manage default builder.
+`
+
+func newBuilderCmd(cfg config, out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "builder",
+		Short: "Manage default builder",
+		Long:  builderCmdHelp,
+		Args:  cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
I wasn't aware of that, thinks for pointing that out. Based on the documentation, it seems that using this would only be useful if we overrode [`UsageFunc`](https://pkg.go.dev/github.com/spf13/cobra?utm_source=godoc#Command.UsageFunc) to provide out own custom output.

I took this approach because other commands in Ketch follow the same `return cmd.Usage()` convention, but should those be changed too? If in the future we were to provide out own custom output I can see a reason for leaving them, but as it stands now it seems like they could all be removed.

Reply:
I think we can skip this one. But if you want it to be there then it is up to you :)

Reply:
I'm cool with removing it since the functionality is built in as you pointed at. Removed.